Apps/WebClient/src/NewClientApp/src/components/private/dependent/AddDependentComponent.vue

```diff
@@ -0,0 +1,457 @@
+                        border
+                    >
+                        <template #text>
+                            <p
+                                data-testid="dependentErrorText"
+                                class="text-body-1"
+                            >
+                                <span v-if="isErrorDataMismatch">
+                                    The information you entered does not match
+                                    our records. Please try again with the exact
+                                    given and last names on the BC Services
+                                    Card.
+                                </span>
+                                <span v-else-if="isErrorNoHdid">
+                                    Please ensure you are using a current
+                                    <a
+                                        href="https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card"
+                                        target="_blank"
+                                        rel="noopener"
+                                        >BC Services Card</a
+                                    >.
+                                </span>
+                                <span v-else-if="isErrorProtected">
+                                    Unable to add dependent.
+                                </span>
+                                <span v-else>{{ errorMessage }}</span>
+                            </p>
+                            <span
+                                v-if="isErrorProtected"
+                                data-testid="condensed-error-contact-message"
+                                class="text-body-1"
+                            >
+                                Please contact
+                                <a href="mailto:HealthGateway@gov.bc.ca"
+                                    >HealthGateway@gov.bc.ca</a
+                                >.
+                            </span>
+                            <span
+                                v-else
+                                data-testid="not-condensed-error-contact-message"
+                                class="text-body-1"
+                            >
+                                If you continue to have issues, please contact
+                                <a href="mailto:HealthGateway@gov.bc.ca"
+                                    >HealthGateway@gov.bc.ca</a
+                                >.
+                            </span>
+                        </template>
+                    </v-alert>
+                    <v-container data-testid="newDependentModalText">
+                        <v-row>
+                            <v-col cols="12" sm="6">
+                                <label for="firstName"
+                                    >First and Middle Names</label
+                                >
+                                <div>
```

Comment:
div is unnecessary here, v-text-field has display: block, thus it takes up the whole row.